parachain-tracer/src/tracker.rs

```diff
@@ -295,6 +300,25 @@ impl SubxtTracker {
 		}
 	}
 
+	fn set_dropped_candidates(&mut self) {
+		for candidates in self.candidates.values_mut() {
+			let mut backed_candidates: Vec<_> = candidates
+				.iter_mut()
+				.flatten()
+				.filter(|candidate| !candidate.is_included())
+				.collect();
+			// We have more than one backed candidate
+			// because of session change or availability core timeout
```

Comment:
It's a good question. 
We could've cleaned them on session change, but we still have to keep in mind that a candidate may be dropped after a core timeout. 
Do you think we should separate that cases?

Reply:
Generally it is not a good idea to build logic on top of side effects. If this fixes it quickly, yeah let's do it, but let's properly handle session changes and otherwise the only reason you could have more than 1 candidate is elastic scaling.